zulipterminal/model.py

```diff
@@ -702,6 +703,55 @@ def get_other_subscribers_in_stream(self, stream_id: Optional[int]=None,
                     if stream['name'] == stream_name
                     if sub != self.user_id]
 
+    def get_tidied_user_info(self, user_id: int) -> Dict[str, Any]:
```

Comment:
I'm not sure if we need "tidied" here, though I appreciate the intent to make it clearer. Potentially using a TypedDict (helper.py perhaps, since it's not an api type) would make it clear to users from the internal method even without an underscore, though the name itself does need to be distinct at runtime of course.